clang/test/SemaSYCL/kernel-arg-opt-report.cpp

```diff
@@ -132,7 +132,7 @@ int main() {
 // SPIR-NEXT: String:          'Arg '
 // SPIR-NEXT: Argument:        '3'
 // SPIR-NEXT: String:          ':'
-// SPIR-NEXT: String:          Compiler generated argument for accessor,
+// SPIR-NEXT: String:          'Compiler generated argument for sycl::accessor<char, 1, sycl::access::mode::read>,'
```

Comment:
The above cases drop the single quote from the new text, but this one adds the single quotes. Why the inconsistency?

Reply:
> The above cases drop the single quote from the new text, but this one adds the single quotes. Why the inconsistency?

@AaronBallman Can't figure out why those quotes are added in one and not in the other. I wonder if it's not the way the remark is emitted in the report. It seems to me that when there are some escape chars, those quotes are added but not sure yet!